app/views/help/cookie_settings.html.haml

```diff
@@ -0,0 +1,59 @@
+- if params[:"analytical-cookies"] == "off"
```

Comment:
Standard: use single quotes unless working with variables, then it is double quotes

Reply:
Done 👍